.travis.yml

```diff
@@ -0,0 +1,201 @@
+---
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+script:
+  # Build docker image
+  - cd 2.6/
+  - docker build -t maestrano/mongo:travis . > /dev/null 2>&1
+
+  # Create volumes
+  - mkdir -p vol-m0 vol-m1 vol-m2
+
+  # Configuration
+  - HOST_PUB_IP=$(ifconfig | grep "eth0 " -A 1 | grep "inet " | cut -d':' -f2 | cut -d' ' -f1)
```

Comment:
I was wondering regarding this example.
Why not 
`ip route get 8.8.8.8 | awk '{print $NF; exit}'`
https://askubuntu.com/questions/430853/how-do-i-find-my-internal-ip-address/604691#604691